howdoi/howdoi.py

```diff
@@ -236,13 +271,25 @@ def _get_answer(args, links):
     text = text.strip()
     return text
 
+def _get_cachable_questions(query):
+    cache_key = query + "-links"
+    res = cache.get(cache_key)
+    if res:
+        if res == "NULL":
```

Comment:
`"NULL"` feels bit gross to me. Is it possible to put `None` directly into the cache? If not, then let's put this as `CACHE_EMPTY_PLACEHOLDER = "NULL"` or similar at the top and use it instead of retyping the string.

Reply:
You can put it in, but you then have no way to distinguish from the key not being present (cache.get() returns None). This is something a good api would account for, but we don't have one. A special Empty class that's bool(res)=true and behaves like None otherwise would be elegant. I don't see the point, but it's your call.

Reply:
OK then let's do the `CACHE_EMPTY_PLACEHOLDER = "NULL"` at the top

Reply:
NULL is a well-known constant used from C to YAML and pickles to something short. I don't see it as any worse than def class EmptyVal -> isinstance(res, EmptyVal), but I don't want to gross you out ;)

Reply:
Yes. what I want to avoid is the string "NULL" in more than one place in the code.

Reply:
4c56e752d3150eab8f8537241877151b8bb59ebd